app/models/github_service.rb

```diff
@@ -9,10 +9,19 @@ def initialize(user)
   end
 
   def repos(user)
-    # require 'pry' ; binding.pry
     parse(connection.get("users/#{user.nickname}/repos"))
   end
 
+  def pull_request_count(user)
+    repos(user).map do |repo|
+      parse(connection.get("repos/#{user.nickname}/#{repo["name"]}/pulls")).count
```

Comment:
Line is too long. [81/80]<br>Prefer single-quoted strings inside interpolations.